ansible/ansible.cfg

```diff
@@ -2,4 +2,3 @@
 host_key_checking = False
 remote_user = ubuntu
 ansible_python_interpreter = /usr/bin/python3
-vault_password_file = "../.vault_password"
```

Comment:
Oh cool, we don't need this anymore?

Reply:
This was added in #259, this is not required.